client.lua

```diff
@@ -783,11 +783,12 @@ function removeBox()
     ClearPedTasks(playerPed)
 end
 
+local boxHash = `hei_prop_heist_box`
 function carryBox()
     postalJobState.isCarryingBox = true
     local ped = PlayerPedId()
     local x,y,z = table.unpack(GetEntityCoords(ped))
-    local prop = CreateObject(GetHashKey('hei_prop_heist_box'), x, y, z + 0.2,  true,  true, true)
+    local prop = CreateObject(boxHash, x, y, z + 0.2,  true,  true, true)
```

Comment:
why do we just throw in `boxHash` here and not `joaat(boxHash)`?

Reply:
The symbol ` allows you to find the hash directly, I put the result in a variable to avoid doing a "calculation" several times with always the same result.

Reply:
So essentially `hei_prop_heist_box` and `GetHashKey('hei_prop_heist_box')` are the same?

Reply:
The result is the same. `GetHashKey` is a native and native calls are known to be "long" in terms of execution time.